### PR TITLE
Use Jetty packaged Jasper and JSTL for compiling JSP instead of Tomcat

### PR DESCRIPTION
**Pull request by jazdw**
Use Jetty packaged Jasper and JSTL for compiling JSP instead of Tomcat 6. Update to Jetty 9.2.11.

Terry, just wanted you to review this when you get back. Its a potentially breaking change but it should be ok.

I've had to include a couple of (bug) patches to make this work. The JSPUtil one I have reported here - https://bz.apache.org/bugzilla/show_bug.cgi?id=58119 and apparently it will be fixed in Tomcat 8.0.25 which will make its way into the Jetty project. Amazingly this has been hanging around for 3 years.

The StandardJarScanner patch is fixed in Jetty 9.3 but we can't use that just yet because its compiled for Java 8.

Basically the purpose of this is to get environment up to date and to enable JSP compiling from Maven.

### CODE REVIEW

**Comment by jazdw** [comment]
This needs infiniteautomation/ma-core-private#2
